### PR TITLE
Add san joaquin

### PR DESCRIPTION
**Pull request by MaddeeRubenson**

### CODE REVIEW

File: data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd [hunk 1]
```diff
@@ -151,11 +151,13 @@ yuba_suit <- suitability |>
...
 
 ## Instream Rearing Habitat
 
-The 1D Hec Ras model was run for for flows 100 through 4500 CFS at all nodes of the bypasses (Chowchilla, Eastside, and Mariposa) and reaches 1, 1A, 2A, and 5. A suitability based on depth and velocity was applied all nodes and flows to get a suitable area for in-channel and floodplain rearing for the Upper San Joaquin River between Merced River and Friant Dam. 
+The 1D Hec Ras model was run for for flows 100 through 4500 CFS at all nodes of the bypasses (Chowchilla, Eastside, and Mariposa) and reaches 1, 1A, 2A, and 5. A suitability criteria based on depth and velocity was applied to all nodes and flows to get a suitable area for in-channel and floodplain rearing for the Upper San Joaquin River between Merced River and Friant Dam.
```
Comment (ErinCain) [inline data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd]: Can we update to active voice? And can you link to the hec ras model or describe what it is here?

File: data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd [hunk 2]
```diff
@@ -151,11 +151,13 @@ yuba_suit <- suitability |>
...
 
 ## Instream Rearing Habitat
 
-The 1D Hec Ras model was run for for flows 100 through 4500 CFS at all nodes of the bypasses (Chowchilla, Eastside, and Mariposa) and reaches 1, 1A, 2A, and 5. A suitability based on depth and velocity was applied all nodes and flows to get a suitable area for in-channel and floodplain rearing for the Upper San Joaquin River between Merced River and Friant Dam. 
+The 1D Hec Ras model was run for for flows 100 through 4500 CFS at all nodes of the bypasses (Chowchilla, Eastside, and Mariposa) and reaches 1, 1A, 2A, and 5. A suitability criteria based on depth and velocity was applied to all nodes and flows to get a suitable area for in-channel and floodplain rearing for the Upper San Joaquin River between Merced River and Friant Dam. 
 
 Suitability criteria was applied to the outputs of the Hec Ras model for suitable depth and velocity for instream juvenile and fry rearing. The Yuba River depth and velocity criteria was used as a proxy for the Upper San Joaquin River using data provided by Mark Gard from the California Department of Fish and Wildlife.
```
Comment (ErinCain) [inline data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd]: What is "The Yuba River" depth and velocity criteria. Please link and describe the document

File: data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd [hunk 3]
```diff
@@ -170,26 +172,27 @@ Reaches used for instream and floodplain rearing can be updated to include mains
...
             max_depth = max(units_si_conv))
 juv_velocity <- yuba_suit |> filter(life_stage == "Juvenile" &  suitability_metric == "Velocity") |> 
-  filter(suitability_index > 0.1 & units_si_conv > 0 ) |> 
+  filter(suitability_index > suit_index & units_si_conv > 0 ) |> 
   summarise(min_vel = min(units_si_conv),
             max_vel = max(units_si_conv))
 
 # reaches to include in juve rearing: all bypasses 
-# TODO: placeholder
 # TODO: check about Connect, Upper, Middle
```
Comment (ErinCain) [inline data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd:206]: Is this still a todo? If not please remove

File: data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd [hunk 4]
```diff
@@ -170,26 +172,27 @@ Reaches used for instream and floodplain rearing can be updated to include mains
 
 
 ```{r message=FALSE, warning=FALSE}
+suit_index <- 0.3
```
Comment (ErinCain) [inline data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd:185]: Can you describe what a suit_index of 0.3 means.

**Review by ErinCain — COMMENTED** [review]
Can you please update the language to be consistently in active voice and add additional details to describe data sources. We can't expect our audience to know what 
"The Yuba River" criteria is. Also please hide the code and just show the plots

**Comment by MaddeeRubenson** [comment]
@ErinCain I've addressed your comments. Please do any necessary additional reviews and merge.

**Pull request merged**